.eslintrc.js

```diff
@@ -19,7 +19,7 @@ module.exports = {
         "id-length": ["error", { "min": 2 }],
         "max-depth": ["error", 4],
         "max-lines": ["error", 200],
-        "complexity": ["error", 2],
+        "complexity": ["error", 5],
```

Comment:
Bumped this since `getJobsFromWorkItems` introduced a bit of logic, including a few conditions (which were added to handle undefined cases for the most part). I may bump this in the future, but will keep an eye on when things get too complex and make reading/digestion harder. The other lint rules like depth and max lines should also start to blow up if things get too gnarly.